### PR TITLE
v93.5.0

### PR DESCRIPTION
**Pull request by romoore**
# 93.5.0

This minor release introduces new APIs for Slider. The track height and the
visibility of track tick marks can be customized.

## New features

`MDCSlider` allows customization of the track height.

**Objective-C**

```objc
MDCSlider *slider = [[MDCSlider alloc] init];
slider.trackHeight = 4;
```

**Swift**

```swift
let slider = MDCSlider()
slider.trackHeight = 4
```

`MDCSlider` also allows controlling the display of the track tick marks for
continuous and discrete sliders.  Setting this property disables the automatic
conversion of a slider to "discrete" when setting `numberOfDiscreteValues`.

**Objective-C**

```objc
MDCSlider *slider = [[MDCSlider alloc] init];
// Track tick marks are always shown. Remains a continuous slider.
slider.trackTickVisibility = MDCSliderTrackTickVisibilityAlways;
slider.minimumValue = 0;
slider.maximumValue = 100;
// Show tick marks at 0, 10, 20, ..., 100
slider.numberOfDiscreteValues = 11;
```

**Swift**

```swift
let slider = MDCSlider()
// Track tick marks are always shown. Remains a continuous slider.
slider.trackTickVisibility = .always
slider.minimumValue = 0;
slider.maximumValue = 100;
// Show tick marks at 0, 10, 20, ..., 100
slider.numberOfDiscreteValues = 11;
```

### ThumbTrack

* [Discrete dots scale slowly with track height. (#8744)](https://github.com/material-components/material-components-ios/commit/8209a86761f81fe9e3b36689608f53cebcff3d40) (Robert Moore)

### Multi-component changes

* [Add API to control showing tick marks. (#8743)](https://github.com/material-components/material-components-ios/commit/826779f2a470a708a95ddfdcba491fe7dd47e5b4) (Robert Moore)
* [Add trackHeight property (#8740)](https://github.com/material-components/material-components-ios/commit/f0647ad0fe20d586599a7c2170982d0d8b645230) (Bryan Oltman)
* [Continuous sliders show tick marks. (#8745)](https://github.com/material-components/material-components-ios/commit/a742a6132e5aa69ce9eec6dcfc2a74bd0c93765e) (Robert Moore)

---

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,45 @@
+# 93.5.0
+
+This minor release introduces new customization APIs for Slider. The track
+height and the visibility of track tick marks can be customized.
+
+## New features
+
+`MDCSlider` allow customization of the track height.
+
+```objc
```
Comment (codeman7) [inline CHANGELOG.md:10]: I believe we traditionally put both obj-c and swift examples.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,45 @@
...
+MDCSlider *slider = [[MDCSlider alloc] init];
+slider.trackHeight = 4;
+```
+
+`MDCSlider` also allows controlling the display of the track tick marks for
+continuous and discrete sliders.  Setting this property disables the automatic
+conversion of a slider to "discrete" when setting `numberOfDiscreteValues`.
+
+```objc
```
Comment (codeman7) [inline CHANGELOG.md:19]: Same

**Review by codeman7 — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (codeman7) [reply CHANGELOG.md:10]: Spoke offline we are going to just do obj-c to help this release land faster.

Comment (romoore) [reply CHANGELOG.md:10]: Done. Looking at the CHANGELOG, it seems we generally don't.

**Review by codeman7 — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (romoore) [reply CHANGELOG.md:19]: Done.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,68 @@
+# 93.5.0
+
+This minor release introduces new APIs for Slider. The track height and the
+visibility of track tick marks can be customized.
+
+## New features
+
+`MDCSlider` allows customization of the track height.
```
Comment (jverkoey) [inline CHANGELOG.md:8]:
```
`MDCSlider` now allows customization
```

To emphasize that this is new functionality, not existing functionality (as presently written).

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,68 @@
...
+```
+
+**Swift**
+
+```swift
+let slider = MDCSlider()
+slider.trackHeight = 4
+```
+
+`MDCSlider` also allows controlling the display of the track tick marks for
```
Comment (jverkoey) [inline CHANGELOG.md:24]:
```
`MDCSlider` now also allows
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,68 @@
...
+**Swift**
+
+```swift
+let slider = MDCSlider()
+// Track tick marks are always shown. Remains a continuous slider.
+slider.trackTickVisibility = .always
+slider.minimumValue = 0;
+slider.maximumValue = 100;
+// Show tick marks at 0, 10, 20, ..., 100
+slider.numberOfDiscreteValues = 11;
```
Comment (jverkoey) [inline CHANGELOG.md:49]: Remove all semicolons.

**Review by jverkoey — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (romoore) [reply CHANGELOG.md:8]: Thanks, Jeff. We spoke offline and agreed that correcting this wasn't worth the trade-off in time for presubmits given the urgent deadline for this Hotfix.

File: CHANGELOG.md (hunk 5 above)
Comment (romoore) [reply CHANGELOG.md:49]: Thanks, Jeff. We spoke offline and agreed that correcting this wasn't worth the trade-off in time for presubmits given the urgent deadline for this Hotfix.

File: CHANGELOG.md (hunk 4 above)
Comment (romoore) [reply CHANGELOG.md:24]: Thanks, Jeff. We spoke offline and agreed that correcting this wasn't worth the trade-off in time for presubmits given the urgent deadline for this Hotfix.

**Pull request merged**